storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -967,12 +967,6 @@
     public static final String STORM_ZOOKEEPER_SUPERACL = "storm.zookeeper.superACL";
 
     /**
-     * The topology Zookeeper authentication scheme to use, e.g. "digest". Defaults to no authentication.
-     */
-    @isString
-    public static final String STORM_ZOOKEEPER_TOPOLOGY_AUTH_SCHEME="storm.zookeeper.topology.auth.scheme";
-
-    /**
```

Comment:
-1 the problem with deleting these is that we don't get any config checks that it is a string.

We could move it to a different file so that it is hidden from end users.